quadra/utils/patch/metrics.py

```diff
@@ -200,8 +203,8 @@ def compute_patch_metrics(
             output_mask = (output_mask > 0).astype(np.uint8)
             gt_img = label(gt_img)
 
-            for i in range(1, gt_img.max() + 1):
-                region = (gt_img == i).astype(bool)
+            for i in range(1, gt_img.max() + 1):  # type: ignore[union-attr]
```

Comment:
before here I think we have need check if 'gt_img' is not None and it is a numpy array. What do you think?